### PR TITLE
Optimize hunyuan3d

### PR DESCRIPTION
**Pull request by STwangyingrui**
Improve Hunyuan3D inference performance with reduced transformer sync overhead, FlashInfer MoE, fused QK RMSNorm, FP8 DiT support, and fused self-attention QKV projection. MoE FP8 remains disabled to avoid accuracy risk.

### CODE REVIEW

File: configs/hunyuan3d/hunyuan3d_shape_fp8.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "octree_resolution": 384,
+    "attn_type": "torch_sdpa",
+    "use_fused_qk_rms_norm": true,
+    "use_fused_qkv_attn": true,
+    "moe_backend": "flashinfer",
+    "moe_flashinfer_setting": {
+        "autotune": true,
+        "tune_max_num_tokens": 8192
+    },
+    "dit_original_ckpt": "/data/nvme0/wangyingrui/Hunyuan3D-2.1-fp8/hunyuan3d-dit-v2-1/model.fp8.safetensors",
```
[the remark this answers is not in the document]
Comment (STwangyingrui) [reply configs/hunyuan3d/hunyuan3d_shape_fp8.json]: fixed.

File: lightx2v/models/networks/hunyuan3d/weights/transformer_weights.py [hunk 2]
```diff
@@ -47,22 +49,86 @@ def __init__(self, config, block_idx, mm_type):
...
+        fc1_biases, fc2_biases = [], []
+        for expert_w in self.experts:
+            fc1_list.append(expert_w.fc1._get_actual_weight().t().contiguous())
+            fc2_list.append(expert_w.fc2._get_actual_weight().t().contiguous())
+            fc1_biases.append(expert_w.fc1._get_actual_bias())
+            fc2_biases.append(expert_w.fc2._get_actual_bias())
+        self._fi_fc1_weight = torch.stack(fc1_list, dim=0)
+        self._fi_fc2_weight = torch.stack(fc2_list, dim=0)
+        self._fi_fc1_bias = self._stack_optional_biases(fc1_biases)
+        self._fi_fc2_bias = self._stack_optional_biases(fc2_biases)
```
[the remark this answers is not in the document]
Comment (STwangyingrui) [reply lightx2v/models/networks/hunyuan3d/weights/transformer_weights.py:132]: fixed.

**Review by helloyongyang — APPROVED** [review]
